web-console/4.0-designs/project-selector/project-selector.md

```diff
@@ -0,0 +1,16 @@
+# Project Selector: Favorites and Defaults
+
+The project selector will be accessible at all times from the secondary masthead.
+
+![image 1](img/1.png)
+![image 2](img/1.1.png)
+
+* The project selector dropdown will have two sections; favorites and defaults will be in the first section of the list and all other projects will be in the second section of the list. The sections will be divided by a horizontal separator.
+* Favorite projects
+  * Favorite projects are above the horizontal separator and have a black (pf-black) pin icon (pficon-thumb-tack) to the left of the project name.
+  * Projects below the separator can be added to the favorites list by clicking its grey (pf-black-300) pin icon.
+  * Favorite projects can be removed from the favorites list by clicking its black pin icon.
+* Default project
+  * The default project is the first project in the list and has a yellow (pf-gold-300) star icon (fa-star).
+  * Users may change the default project by clicking the grey (pf-black-300) star icon next to any of their favorite projects.
+  * Only one project may be the default project at any given time.
```

Comment:
Can you explain what a "default project" is, since it may not be clear from this page.